### PR TITLE
 First version changing variables names

### PR DESCRIPTION
**Pull request by ricardoul**
No merge before @t-j-turner makes the changes to the env variables

### CODE REVIEW

**Comment by vitorpy** [comment]
This pull request **introduces 1 alert** when merging 9444cad04079b43b48b2bd1c0d26f5240f51b93d into 85be13183f832f7a1459481d119e76e197a95eb9 - [view on LGTM.com](https://lgtm.com/projects/g/pillarwallet/pillar-bcx-pubsub/rev/pr-3835c29dbddd25cea28ff7ad3e9db94391db7528)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/controllers/accounts_ctrl.js [hunk 1]
```diff
@@ -24,8 +24,9 @@ const logger = require('../utils/logger.js');
 
 const accounts = require('../models/accounts_model');
 const mongoose = require('mongoose');
+const config = require("../config/")
```
Comment (IAmKio) [inline src/controllers/accounts_ctrl.js]: Don't forget the tailing `;`

File: src/controllers/accounts_ctrl.js [hunk 2]
```diff
@@ -24,8 +24,9 @@ const logger = require('../utils/logger.js');
...
 
-const ACCOUNTS_NUMBER_TO_FETCH = process.env.ACCOUNTS_NUMBER_TO_FETCH ? process.env.ACCOUNTS_NUMBER_TO_FETCH : 100;
+const ACCOUNTS_NUMBER_TO_FETCH = config.get("accounts.numberToFetch")
```
Comment (IAmKio) [inline src/controllers/accounts_ctrl.js]: Here too: `;`

File: src/housekeeper.js [hunk 3]
```diff
@@ -31,38 +31,34 @@ const logger = require('./utils/logger');
 const dbServices = require('./services/dbServices');
 const fs = require('fs');
 const abiPath = `${require('app-root-path')}/src/abi/`;
+const config = require('./config/index')
```
Comment (IAmKio) [inline src/housekeeper.js]: Here too: `;`

File: src/master.js [hunk 4]
```diff
@@ -26,13 +26,8 @@ require('./utils/diagnostics');
 const logger = require('./utils/logger');
 const { fork } = require('child_process');
 
-const optionDefinitions = [
-  { name: 'protocol', alias: 'p', type: String },
-  { name: 'maxWallets', type: Number },
-];
-const commandLineArgs = require('command-line-args');
+const config = require("./config")
```
Comment (IAmKio) [inline src/master.js]: And here: `;`

**Comment by IAmKio** [comment]
@ricardoul just a note on force-pushing:
https://developer.atlassian.com/blog/2015/04/force-with-lease/

File: README.md [hunk 5]
```diff
@@ -8,14 +8,13 @@ When the application is launched in a deployed environment we make no assumption
...
-- GETH_NODE_PORT - port to open a websocket on
-- CHECKSUM_KEY ???
-- HASH_PREFIX ???
+- BCX_MONGO_SERVER_ADDRESS (mongo db url)
+- BCX_MONGO_USER
+- BCX_MONGO_PASSWORD
+- BCX_MONGO_DATABASE_NAME
+- BCX_GETH_NODE_URL  - url of a geth node for the app to talk to
+- BCX_GETH_NODE_PORT - port to open a websocket on
+- BCX_CHECKSUM_KEY ???
```
Comment (IAmKio) [inline README.md]: @t-j-turner has pointed out that this README which they are following contains variables that are missing. Could you please split this out by service (such as housekeeper, deferred etc) and what environment variables are required for each? Thanks!

**Comment by IAmKio** [comment]
@t-j-turner could you have a peek and check you're okay with the YML files?

**Pull request merged**